log10/_httpx_utils.py

```diff
@@ -29,6 +30,24 @@
 httpx_async_client = httpx.AsyncClient(timeout=timeout)
 
 
+class LLM_PROVIDER(Enum):
+    ANTHROPIC = "Anthropic"
+    OPENAI = "OpenAI"
+    UNKNOWN = "Unknown"
+
+
+PROVIDER_PATHS = {
+    LLM_PROVIDER.ANTHROPIC: ["/v1/messages", "/v1/complete"],
+    LLM_PROVIDER.OPENAI: ["v1/chat/completions", "chat/completions"],
+}
+
+USER_AGENT_NAME_TO_PROVIDER = {
+    "AsyncOpenAI": LLM_PROVIDER.OPENAI,
```

Comment:
It's not used for now. I wonder if we should add `"OpenAI": LLM_PROVIDER.OPENAI,` here?